Resources/Private/JavaScript/media-module/src/components/SideBarRight/Inspector/AssetInspector.tsx

```diff
@@ -80,8 +84,7 @@ const AssetInspector = () => {
     }, [selectedAsset?.id]);
 
     if (!selectedAsset || selectedInspectorView !== 'asset') return null;
-
-    return (
+    const PropertyInspector = () => (
```

Comment:
We should probably extract the property inspector into its own component.

Reply:
I was a bit reluctant with that, because i wasn't sure, how the tab idea would turn out...but i moved it now